src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -200,482 +198,306 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitDescribeQuery(
-      SparqlAutomaticParser::DescribeQueryContext* ctx) override {
+  antlrcpp::Any visitDescribeQuery(Parser::DescribeQueryContext* ctx) override {
+    // TODO: unsupported
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitAskQuery(
-      SparqlAutomaticParser::AskQueryContext* ctx) override {
+  antlrcpp::Any visitAskQuery(Parser::AskQueryContext* ctx) override {
+    // TODO: unsupported
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitDatasetClause(
-      SparqlAutomaticParser::DatasetClauseContext* ctx) override {
+  antlrcpp::Any visitDatasetClause(Parser::DatasetClauseContext* ctx) override {
+    // TODO: unsupported
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitDefaultGraphClause(
-      SparqlAutomaticParser::DefaultGraphClauseContext* ctx) override {
+      Parser::DefaultGraphClauseContext* ctx) override {
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitNamedGraphClause(
-      SparqlAutomaticParser::NamedGraphClauseContext* ctx) override {
+      Parser::NamedGraphClauseContext* ctx) override {
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitSourceSelector(
-      SparqlAutomaticParser::SourceSelectorContext* ctx) override {
+      Parser::SourceSelectorContext* ctx) override {
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitWhereClause(
-      SparqlAutomaticParser::WhereClauseContext* ctx) override {
+  antlrcpp::Any visitWhereClause(Parser::WhereClauseContext* ctx) override {
     return visitChildren(ctx);
   }
 
   antlrcpp::Any visitSolutionModifier(
-      SparqlAutomaticParser::SolutionModifierContext* ctx) override {
+      Parser::SolutionModifierContext* ctx) override {
     return visitChildren(ctx);
```

Comment:
The default call to `visitChildren` is always a bad idea. Make it explicit by either throwing "NotSupported", or by calling 
`visit` on the single child explicitly.